src/lightning/pytorch/CHANGELOG.md

```diff
@@ -65,6 +65,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Fixed broadcast at initialization in `MPIEnvironment` ([#19074](https://github.com/Lightning-AI/lightning/pull/19074))
 
 
+- Fixed NeptuneLogger lack of usage of `step` parameter when logging metrics ([#19126](https://github.com/Lightning-AI/pytorch-lightning/pull/19126))
```

Comment:
We would have to move this to the Changed section, since the previous behavior was intentionally chosen like this (as pointed out in the comment in the source code).

Reply:
You're right. Done 😉